### PR TITLE
add ansible-tower1 to patch Tuesday

### PR DESCRIPTION
**Pull request by kayiwa**
we had ansible-tower and any redhat based OSes to be eligible for patch
tuesday

### CODE REVIEW

**Comment by kayiwa** [comment]
closes #3633

File: playbooks/os_updates.yml [hunk 1]
```diff
@@ -63,20 +63,46 @@
...
+        name: "*" 
+        state: latest
+        update_cache: true
+        update_only: true
+      register: yum_update_status
+      when: ansible_os_family == "RedHat"
+
+    - name: RedHat | remove unneeded packages
+      ansible.builtin.yum:
+        autoremove: true
```
Comment (acozine) [inline playbooks/os_updates.yml:84]: Oh, interesting, does this work without a package list?

File: playbooks/os_updates.yml [hunk 2]
```diff
@@ -63,20 +63,46 @@
...
+      ansible.builtin.reboot:
+        msg: "Reboot initiated by Ansible because of yum updates."
+        connect_timeout: 5
+        reboot_timeout: 600
+        pre_reboot_delay: 0
+        post_reboot_delay: 15
+        test_command: whoami
+      when: 
+        - ansible_os_family == "RedHat"
+        - yum_update_status.changed
```
Comment (acozine) [inline playbooks/os_updates.yml:103]: We need to register something here and then add a task to reboot the CentOS/RH boxes when required.

**Review by acozine — COMMENTED** [review]
This is a great addition. There are a few bits to polish.

File: playbooks/os_updates.yml (hunk 1 above)
Comment (kayiwa) [reply playbooks/os_updates.yml:84]: If I read this right. https://docs.ansible.com/ansible/latest/collections/ansible/builtin/yum_module.html#parameter-autoremove

File: playbooks/os_updates.yml (hunk 2 above)
Comment (kayiwa) [reply playbooks/os_updates.yml:103]: I thought reboot happens when this register status changes. https://github.com/pulibrary/princeton_ansible/blob/be341d4f54912eb8c6254bbeb34e1596cfaa15a7/playbooks/os_updates.yml#L79

Comment (acozine) [reply playbooks/os_updates.yml:103]: Oh! You're absolutely right. I was reading the `name:` of the task and not what the task actually does.

File: playbooks/os_updates.yml [hunk 3]
```diff
@@ -63,20 +63,46 @@
...
+      when: ansible_os_family == "RedHat"
+
+    - name: Ubuntu | Check if reboot required
       ansible.builtin.stat:
         path: /var/run/reboot-required
       register: reboot_required_file
       when: ansible_os_family == "Debian"
 
-    - name: os_updates | Reboot if required
+    - name: RedHat | Check if reboot required
```
Comment (acozine) [inline playbooks/os_updates.yml:93]:
```suggestion
    - name: RedHat | Reboot if required
```

**Pull request merged**